core/api/views/replenishment.py

```diff
@@ -317,6 +318,35 @@ def get(self, request, *args, **kwargs):
             .order_by("name")
         ]
 
+        # TODO: I think this might return duplicates if a country has multiple
+        # statuses
```

Comment:
To avoid dupicates and to make the queries more readable (they don't specifically need to be the fastest in this case) you could use the `ceit_countries_qs` defined a few lines below, by moving it above and using here a filter like:
`country_id__in=ceit_countries_qs.values_list("id", flat=True)`

Reply:
Done :D